src/app/shared-services/location/location.service.ts

```diff
@@ -6,17 +6,21 @@ import { SqliteService } from './../db/sqlite.service'
 import { InferenceService } from './../inferences/inference.service'
 import { NotificationService } from './../notification/notification.service'
 import { NotificationType } from './../notification/types'
-import { Plugins } from '@capacitor/core'
 import { TranslateService } from '@ngx-translate/core'
 import { LogfileService } from '../logfile/logfile.service'
 import { LogEventScope, LogEventType } from '../logfile/types'
 import {
   BackgroundGeolocationPlugin,
   Location,
 } from '@capacitor-community/background-geolocation'
-
 // eslint-disable-next-line @typescript-eslint/naming-convention
-const { App, BackgroundGeolocation } = Plugins
+const BackgroundGeolocation = registerPlugin<BackgroundGeolocationPlugin>(
+  'BackgroundGeolocation'
+)
```

Comment:
I would prefer this to sit below the imports instead of in between. What do you think?

Reply:
I agree: f678643d9251dd9f6fc879fe3564dc48ce40b1f9